### PR TITLE
Analys och utredning

### PR DESCRIPTION
**Pull request by amawre**

### CODE REVIEW

File: html/analys_och_utredning.html [hunk 1]
```diff
@@ -34,12 +34,34 @@
...
+
+            Avdelningen ansvarar för sammanställningar av Konstnärsnämndens styrdokument, årsredovisning, budgetunderlag, remisser och yttranden, gör uppföljningar och utvärderingar samt skapar rutiner för dessa av Konstnärsnämndens verksamhet.
+
+            <h2>Samverkan med myndigheter och ideella organisationer</h2>
+            <p>Konstnärsnämnden arbetar kontinuerligt med att informera om konstnärers villkor och personalen inbjuds som experter till seminarier och debatter och informationsmöten. Konstnärsnämnden har inom uppdraget att bevaka konstnärernas villkor återkommande möten med bland annat Kulturdepartementet, Riksdagens kulturutskott och myndigheter som Statens Kulturråd, Statens Musikverk, Statens Konstråd och Myndigheten för kulturanalys.
+
+            Konstnärsnämnden träffar också andra institutioner och organisationer som KLYS, enskilda konstnärsorganisationer, konstnärliga högskolor, Centrumbildningar, Scenkonstallianser med flera.
+
+            För att följa konstnärernas ekonomiska och sociala situation har Konstnärsnämnden också inlett en myndighetsdialog med Försäkringskassan, Skatteverket och AF Kultur och Medias ledning.</p>
+
```
Comment (sandra-wallen) [inline html/analys_och_utredning.html:57]: Could this be marked up further?

**Review by sandra-wallen — CHANGES_REQUESTED** [review]
Let's discuss my comment at friday meeting :)

File: html/analys_och_utredning.html [hunk 2]
```diff
@@ -34,12 +34,34 @@
...
+            <li><a href="/index.html">Startsida</a> / </li>
+            <li>Analys och utredning</li>
+        </ul>
+        <section>
+            <h1>Analys och utredning</h1>
 
+            <p>Analysavdelningen hanterar konstnärspolitiska och kulturpolitiska frågor och håller sig underrättad om konstnärernas sociala och ekonomiska villkor samt förutsättningar för konstnärligt skapande. Det innebär bland annat att löpande bevaka trygghetssystemens utformning och tillämpning i förhållande till konstnärlig verksamhet.
+            Det sker genom statistiska och kvalitativa undersökningar, information genom <a href="#">Konstnärsguiden</a> samtalsplattformar med andra politikområden och möten med andra myndigheter och organisationer.
+
+            Under senare år har Konstnärsnämnden roll som samverkanspart i olika regeringsuppdrag och som remissinstans förstärkts. Utgångspunkten för detta arbete är att se frågorna ur konstnärernas perspektiv. Det handlar också om att bevaka konstnärsfrågorna på EU- och internationell nivå.
```
Comment (RicardoCastelbon) [inline html/analys_och_utredning.html:47]: That should be in its own p?

Comment (RicardoCastelbon) [inline html/analys_och_utredning.html:47]: This should be in its own p? It's a new paragraph isn't it?

File: html/analys_och_utredning.html [hunk 3]
```diff
@@ -34,12 +34,34 @@
...
+
+            Avdelningen ansvarar för sammanställningar av Konstnärsnämndens styrdokument, årsredovisning, budgetunderlag, remisser och yttranden, gör uppföljningar och utvärderingar samt skapar rutiner för dessa av Konstnärsnämndens verksamhet.
```
Comment (RicardoCastelbon) [inline html/analys_och_utredning.html:49]: This should be in its own p? It's a new paragraph isn't it?

Comment (RicardoCastelbon) [reply html/analys_och_utredning.html:49]: Same with the rest

**Review by amawre — COMMENTED** [review]
Looks great!

**Pull request merged**